lib/auth/new_web_user.go

```diff
@@ -220,9 +220,9 @@ func (s *AuthServer) CreateUserWithToken(token string, password string, otpToken
 }
 
 func (s *AuthServer) CreateUserWithU2FToken(token string, password string, response u2f.RegisterResponse) (services.WebSession, error) {
-	err := s.CheckU2FEnabled()
+	_, err := s.GetUniversalSecondFactor()
```

Comment:
What's the point of calling `GetXXX()` type of a function and ignoring the result? "getters" aren't supposed to change state, or at least that's what a code reader may expect. Is it changing state in this case? Maybe the function should be renamed? At the very least a comment should cover this strange block.